gliderpy/plotters.py

```diff
@@ -18,21 +18,23 @@
 
 def plot_track(df):
     """
-    Plots a track of glider path coloured by temperature
+    Plots a track of glider path colored by temperature
     :return: figures, axes
     """
-
-    x = df["longitude (degrees_east)"]
-    y = df["latitude (degrees_north)"]
+    x = df["longitude"]
+    y = df["latitude"]
     dx, dy = 2, 4
 
     fig, ax = plt.subplots(
         figsize=(9, 9),
         subplot_kw={"projection": ccrs.PlateCarree()},
     )
-    ax.scatter(x, y, c=None, s=25, alpha=0.25, edgecolor="none")
+    # Add a label to the scatter plot
```

Comment:
When the code is clear we don't need comments describing what it does as the code may change quicker and the comment may become stale and confusing.

```suggestion
```